near-sdk/src/environment/env.rs

```diff
@@ -698,6 +703,18 @@ pub fn storage_has_key(key: &[u8]) -> bool {
         _ => panic!(RETURN_CODE_ERR),
     }
 }
+/// Accessing receipts created by the contract. Only available in unit tests.
+pub fn created_receipts() -> Vec<Receipt> {
```

Comment:
```suggestion
#[cfg(test)]
pub fn created_receipts() -> Vec<Receipt> {
```

Reply:
This isn't going to work, because `#[cfg(test)]` only works for local module. So if you are using a library and want to use test-only method, it's not going to be available within your tests.